src/googletest.h

```diff
@@ -75,14 +75,21 @@ _END_GOOGLE_NAMESPACE_
 #undef GLOG_EXPORT
 #define GLOG_EXPORT
 
+static vector<string>* temp_directories_list;
+
 static inline string GetTempDir() {
-#ifndef GLOG_OS_WINDOWS
-  return "/tmp";
-#else
-  char tmp[MAX_PATH];
-  GetTempPathA(MAX_PATH, tmp);
-  return tmp;
-#endif
+  if (temp_directories_list == NULL) {
+    temp_directories_list = new vector<string>;
+    google::GetExistingTempDirectories(temp_directories_list);
+
+    if (temp_directories_list->empty()) {
+      fprintf(stderr, "No temporary directory found\n");
+      exit(1);
```

Comment:
```suggestion
      exit(EXIT_FAILURE);
```
`EXIT_FAILURE` is defined `<cstdlib>`.

Reply:
OK. I will also update (in separate commit) other instances in glog source code:
```
$ grep -r 'exit(' *
README.rst:     exit(1);
src/logging_unittest.cc:    exit(0);
src/logging_custom_prefix_unittest.cc:    exit(0);
src/googletest.h:      exit(1);
src/googletest.h:      exit(1);                                                                 \
src/googletest.h:      exit(1);                                          \
src/googletest.h:      exit(1);                                                          \
src/googletest.h:      exit(1);                                                  \
src/googletest.h:      exit(1);                                                  \
src/googletest.h:      exit(1);                                                          \
src/googletest.h:      exit(1);                                                          \
src/googletest.h:      exit(1);                                                          \
src/glog/raw_logging.h.in:    exit(1); \
src/glog/logging.h.in:  @ac_cv___attribute___noreturn@ ~NullStreamFatal() throw () { _exit(1); }
```

Reply:
Thanks!